src/utils/getConfigsForFile.ts

```diff
@@ -2,22 +2,24 @@ import * as path from 'path';
 import Config from '../types/config/Config';
 import NormalizedPath from '../types/NormalizedPath';
 import normalizePath from './normalizePath';
-import getAllConfigs from './getAllConfigs';
+import getConfigManager from './getConfigManager';
 
-// Returns an array of all the configs that apply to a given file
-export default function getConfigsForFile(filePath: NormalizedPath): Config[] {
-    let allConfigs = getAllConfigs();
-    let configsForFile: Config[] = [];
+export function getConfigPathCandidatesForFile(filePath: NormalizedPath): string[] {
```

Comment:
Nit: can you split this out into a separate file?  As coded there's a cycle between this file and `getConfigManager`...which isn't strictly a problem, but is easy to resolve.